frontend/src/components/DataCard/SuggestedLabels.jsx

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
-export default function SuggestedLabels({ card, labels }) {
-    if (labels.length <= 5) {
+export default function SuggestedLabels({ card, labels, onSelectLabel }) {
+    const [suggestions, setSuggestions] = useState();
+
+    useEffect(() => {
+        fetch(`/api/comparisons/${card.text.project}?text=${card.text.text}`)
+            .then(res => res.json().then(result => setSuggestions(result)));
+    }, []);
+
+    if (labels.length <= 5 || !suggestions) {
         return null;
     }
 
     return (
         <div className="suggestions">
             <h4>Suggested Labels</h4>
```

Comment:
Tiny thing, but right now the boxes for suggested labels and background data are exactly the same style. Whenever a new card loads, it takes a sec to remember which is which. I'd recommend we do something like change the shading or outline on one of them so they stand out.